src/overlays/actors/ovl_Dm_Al/z_dm_al.c

```diff
@@ -30,24 +27,133 @@ const ActorInit Dm_Al_InitVars = {
     (ActorFunc)DmAl_Draw,
 };
 
-#endif
-
-extern UNK_TYPE D_0600A0D8;
-
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/ovl_Dm_Al/func_80C1BD90.s")
-
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/ovl_Dm_Al/func_80C1BDD8.s")
-
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/ovl_Dm_Al/DmAl_Init.s")
-
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/ovl_Dm_Al/DmAl_Destroy.s")
-
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/ovl_Dm_Al/DmAl_Update.s")
-
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/ovl_Dm_Al/func_80C1C028.s")
-
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/ovl_Dm_Al/func_80C1C064.s")
-
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/ovl_Dm_Al/func_80C1C11C.s")
+static AnimationInfoS sAnimationInfos[] = {
+    { &object_al_Anim_00DBE0, 1.0f, 0, -1, ANIMMODE_LOOP, 0 },
+};
 
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/ovl_Dm_Al/DmAl_Draw.s")
+s32 func_80C1BD90(DmAl* this, s32* animationIndex) {
```

Comment:
```suggestion
s32 func_80C1BD90(DmAl* this, s32 animationIndex) {
```
Second arg is not a pointer